shared/src/androidMain/kotlin/com/mbta/tid/mbta_app/repositories/AccessibilityStatusRepository.android.kt

```diff
@@ -11,9 +12,17 @@ class AccessibilityStatusRepository(context: Context) : IAccessibilityStatusRepo
     // https://stackoverflow.com/a/59950182
     override fun isScreenReaderEnabled(): Boolean {
         if (accessibility == null) return false
-        return accessibility.isEnabled &&
+        if (!accessibility.isEnabled) return false
+        val services =
             accessibility
                 .getEnabledAccessibilityServiceList(AccessibilityServiceInfo.FEEDBACK_ALL_MASK)
-                .isNotEmpty()
+                .filter {
+                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+                        it.isAccessibilityTool
+                    } else {
+                        it.feedbackType != AccessibilityServiceInfo.FEEDBACK_GENERIC
+                    }
```

Comment:
My non-accessibility accessibility services both report FEEDBACK_GENERIC and no other types of feedback, but TalkBack reports several values here (I think FEEDBACK_SPOKEN, FEEDBACK_HAPTIC, and one other thing).